source/sass/blocks/modal.scss

```
@@ -1,6 +1,44 @@
 .modal {
   display: none;
-  padding: 44px 22px 52px 22px;
+  position: fixed;
+  top: 30%;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.modal__container {
+  width: 320px;
+  margin: 0 auto;
+  background: $color-white;
+  border-radius: 5px;
+  text-align: center;
+  box-shadow: 0, 7px, 15px 0, #00010126;
```

Параметры тени без запятых пишутся. Через запятую можно вторую тень прописать (как с фоном)
```suggestion
  box-shadow: 0 7px 15px 0 #00010126;
```